Common/Api/Backtest.cs

```diff
@@ -78,6 +72,44 @@ public class Backtest : RestResponse
         /// </summary>
         [JsonProperty(PropertyName = "created")]
         public DateTime Created;
+
+        /// <summary>
+        /// Contains population averages scores over the life of the algorithm
+        /// </summary>
+        [JsonProperty(PropertyName = "alphaRuntimeStatistics", NullValueHandling = NullValueHandling.Ignore)]
+        public AlphaRuntimeStatistics AlphaRuntimeStatistics;
+
+        /// <summary>
+        /// Charts updates for the live algorithm since the last result packet
+        /// </summary>
+        [JsonProperty(PropertyName = "charts", NullValueHandling = NullValueHandling.Ignore)]
+        public IDictionary<string, Chart> Charts;
+
+        /// <summary>
+        /// Statistics information sent during the algorithm operations.
+        /// </summary>
+        /// <remarks>Intended for update mode -- send updates to the existing statistics in the result GUI. If statistic key does not exist in GUI, create it</remarks>
+        [JsonProperty(PropertyName = "statistics", NullValueHandling = NullValueHandling.Ignore)]
+        public IDictionary<string, string> Statistics;
+
+        /// <summary>
+        /// Runtime banner/updating statistics in the title banner of the live algorithm GUI.
+        /// </summary>
+        [JsonProperty(PropertyName = "runtimeStatistics", NullValueHandling = NullValueHandling.Ignore)]
+        public IDictionary<string, string> RuntimeStatistics;
```

Comment:
Just checking: there are some properties that the `BacktestResult` has that are now not present, should we add them? like for example `Orders/OrderEvents/ProfitLoss/RollingWindow/...`

Reply:
Order/OrderEvents is not in the Backtest response so we don't have to worry about that one.

I went ahead and added RollingWindow and TotalPerformance so those are good to go.

ProfitLoss is having some issues because of a strange inconsistency in the web api. If there is no entries in the profitloss object the web api send a empty list, if there is, it sends a dictionary type (which is expected). Newtonsoft isn't happy about the empty list though when it expects a dictionary and fails to de-serialize.